packages/@ourworldindata/grapher/src/stackedCharts/StackedAreaChart.tsx

```diff
@@ -663,7 +604,7 @@ export class StackedAreaChart
                     <StackedAreas
                         dualAxis={dualAxis}
                         seriesArr={series}
-                        focusedSeriesName={this.hoveredSeriesName}
+                        hoveredSeriesName={this.hoveredSeriesName}
```

Comment:
Just wondering, what's this change about/for? For thumbnails, it looks like what we care about is focus, not hovering?